tardis/model/tests/test_csvy_model.py

```diff
@@ -34,24 +42,142 @@ def test_compare_models(filename):
     for prop in config_model_props:
         csvy_model_val = csvy_model.get_properties()[prop]
         config_model_val = config_model.get_properties()[prop]
-        if prop == 'homologous_density':
-            npt.assert_array_almost_equal(csvy_model_val.density_0.value, config_model_val.density_0.value)
-            npt.assert_array_almost_equal(csvy_model_val.time_0.value, config_model_val.time_0.value)
+        if prop == "homologous_density":
+            npt.assert_array_almost_equal(
+                csvy_model_val.density_0.value, config_model_val.density_0.value
+            )
+            npt.assert_array_almost_equal(
+                csvy_model_val.time_0.value, config_model_val.time_0.value
+            )
         else:
-            if hasattr(config_model_val, 'value'):
+            if hasattr(config_model_val, "value"):
                 config_model_val = config_model_val.value
                 csvy_model_val = csvy_model_val.value
             npt.assert_array_almost_equal(csvy_model_val, config_model_val)
 
+    assert csvy_model.raw_abundance.shape == config_model.raw_abundance.shape
+    assert (
+        csvy_model.raw_isotope_abundance.shape
+        == config_model.raw_isotope_abundance.shape
+    )
+    assert csvy_model.abundance.shape == config_model.abundance.shape
+    npt.assert_array_almost_equal(
+        csvy_model.raw_abundance.to_numpy(), config_model.raw_abundance.to_numpy()
+    )
+    npt.assert_array_almost_equal(
+        csvy_model.raw_isotope_abundance.to_numpy(),
+        config_model.raw_isotope_abundance.to_numpy(),
+    )
+    npt.assert_array_almost_equal(
+        csvy_model.abundance.to_numpy(), config_model.abundance.to_numpy()
+    )
+
 
-def test_csvy_abundance():
-    csvypath = os.path.join(DATA_PATH, 'config_v_filter.yml')
+@pytest.fixture(scope="module")
+def csvy_model_abundances():
```

Comment:
Add a docstring.  This function has a confusing name.  Is it supposed to return a Radial1DModel object or only the abundances dataframe?